apps/ledger-live-desktop/src/newArch/features/Collectibles/Ordinals/components/Inscriptions/DetailsDrawer/index.tsx

```diff
@@ -1,18 +1,66 @@
 import React from "react";
-import { SideDrawer } from "~/renderer/components/SideDrawer";
 import { SimpleHashNft } from "@ledgerhq/live-nft/api/types";
+import { DetailDrawer } from "LLD/features/Collectibles/components";
+import { CollectibleTypeEnum } from "LLD/features/Collectibles/types/enum/Collectibles";
+import { Flex, Icons, Text } from "@ledgerhq/react-ui";
+import Button from "~/renderer/components/Button";
+import useInscriptionDetailDrawer from "./useInscriptionDetailDrawer";
+import { t } from "i18next";
+
+type ViewProps = ReturnType<typeof useInscriptionDetailDrawer> & {
+  onClose: () => void;
+};
 
 type Props = {
   inscription: SimpleHashNft;
   onClose: () => void;
+  isLoading: boolean;
 };
-const InscriptionDetailsDrawer: React.FC<Props> = ({ inscription, onClose }) => {
-  // will be replaced by DetailsDrawer from collectibles
-  return (
-    <SideDrawer direction={"left"} isOpen={!!inscription} onRequestClose={onClose}>
-      {inscription.name || inscription.contract.name}
-    </SideDrawer>
-  );
+
+const View: React.FC<ViewProps> = ({ data, onClose }) => (
+  <DetailDrawer
+    collectibleType={CollectibleTypeEnum.NFT}
+    areFieldsLoading={data.areFieldsLoading}
+    collectibleName={data.collectibleName}
+    contentType={data.contentType}
+    collectionName={data.collectionName}
+    details={data.details}
+    previewUri={data.previewUri}
+    originalUri={data.originalUri}
+    isPanAndZoomOpen={data.isPanAndZoomOpen}
+    mediaType={data.mediaType}
+    tags={data.tags}
+    useFallback={data.useFallback}
+    tokenId={data.tokenId}
```

Comment:
{...data} will be cleaner :)

Reply:
I've a small remark about this review:  @mcayuelas-ledger  / @LucasWerey 
spreading data assumes that all the object properties are expected from `DetailDrawer` component's api (or contract), if you do so  then you should strictly type the data props and why not type-check (with zod)  so that you guarantee that everything spread satisfies the component props and to avoid that any changes (for example accidently injecting `dangerouslySetInnerHTML` prop) on the passed `data`props will be handled. 
In the meanwhile, there is a rule called commonly used in react code guidelines `react/jsx-props-no-spreading`, this rule don't recommend making spread inside the jsx , instead you can deconstruct the data object before the return operation. With this approach 
- it's more readable (i don't need to access to the component and do reverse engineering to understand what's passed by data and what's not)
- You make sure what to extract from data. (exactly like you ve done, but with less props chaining access operation via data `data.prop`.

Both ways leads to the same result no doubt.

Reply:
I've reverted to previous version without the spread operator